database-factory/dbfactory.go

```diff
@@ -41,6 +46,10 @@ func CreateCluster(cluster *model.Cluster) error {
 			return errors.Wrap(err, "failed to run Terraform apply")
 		}
 		logger.Info("successfully applied Terraform template")
+		err = sendMattermostNotification(cluster, "The Database Factory successfully deployed a new RDS Aurora cluster")
+		if err != nil {
+			return errors.Wrap(err, "failed tο send Mattermost notification")
```

Comment:
Does it make sense to return an error here? The cluster would still get created but we'd just miss the notification, right? Wondering if just logging the error like you do above would be good enough

Reply:
Yeap, I agree on that. The functionality we want to achieve is the creation of the RDS. If the notification fails it should just log this out. FIxed ;)